canonical-web-and-design/maas-ui
# Add the main MAAS nav

Done:
- Fetch the authenticated user details over the websocket. Block loading the app on getting that user.
- Add the main MAAS nav, same as the existing one. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1337. 

QA:
- Make sure your `config.js` has all the fields from `config.example.js`.
- `./run`
- See that the header is the same as MAAS.
- Clicking on anything but Settings should take you to the MAAS set in your config.js.
